src/Integration.Vsix/SonarLintTagger/TaggerProvider.cs

```diff
@@ -196,6 +196,12 @@ public void RequestAnalysis(string path, string charset, IEnumerable<AnalysisLan
                             analyzerOptions, cancellationToken),
                     analysisTimeout);
             }
+            catch (NotSupportedException ex)
+            {
+                // Display a simple user-friendly message for options we know are not supported.
+                // See https://github.com/SonarSource/sonarlint-visualstudio/pull/2212
+                logger.WriteLine($"Analysis error: {ex.Message}");
```

Comment:
Perhaps we should rephrase it not as an error, e.g. "Unsupported analysis option: ..."
WDYT?

Reply:
That doesn't make it clear that the analysis hasn't taken place (could be interpreted as meaning the setting is not taken into account).

I'll change it to "Unable to analyze: ..."